jbpm-designer-backend/src/test/java/org/jbpm/designer/test/bpmn2/Bpmn2UnmarshallingTestCase.java

```diff
@@ -48,17 +49,18 @@ private static File getTestJsonFile(String filename) {
         return new File(fileURL.getFile());
     }
 
-    @Test
-    public void testSimpleDefinitionsUnmarshalling() throws Exception {
-        Bpmn2JsonUnmarshaller unmarshaller = new Bpmn2JsonUnmarshaller();
-        Definitions definitions = ((Definitions) unmarshaller.unmarshall(getTestJsonFile("empty.json"), "").getContents().get(0));
-        assertEquals("<![CDATA[my doc]]>", definitions.getDocumentation().iterator().next().getText());
-        assertEquals("http://www.w3.org/1999/XPath", definitions.getExpressionLanguage());
-        assertEquals("http://www.omg.org/bpmn20", definitions.getTargetNamespace());
-        assertEquals("http://www.w3.org/2001/XMLSchema", definitions.getTypeLanguage());
-        assertTrue(definitions.getRootElements().size() == 1);
-        definitions.eResource().save(System.out, Collections.emptyMap());
-    }
+    // disabling
+//    @Test
+//    public void testSimpleDefinitionsUnmarshalling() throws Exception {
+//        Bpmn2JsonUnmarshaller unmarshaller = new Bpmn2JsonUnmarshaller();
+//        Definitions definitions = ((Definitions) unmarshaller.unmarshall(getTestJsonFile("empty.json"), "").getContents().get(0));
+//        assertEquals("<![CDATA[my doc]]>", definitions.getDocumentation().iterator().next().getText());
+//        assertEquals("http://www.w3.org/1999/XPath", definitions.getExpressionLanguage());
+//        assertEquals("http://www.omg.org/bpmn20", definitions.getTargetNamespace());
+//        assertEquals("http://www.w3.org/2001/XMLSchema", definitions.getTypeLanguage());
+//        assertTrue(definitions.getRootElements().size() == 1);
+//        definitions.eResource().save(System.out, Collections.emptyMap());
+//    }
```

Comment:
I guess we have to remove tests for not implemented functionality for 6.4.X branch. In my opinion there are no reasons to store it here in this way.